plugins/plugin-maven/che-plugin-maven-server/src/main/java/org/eclipse/che/plugin/maven/server/core/MavenWorkspace.java

```diff
@@ -110,9 +110,16 @@ public void projectUpdated(Map<MavenProject, MavenProjectModifications> updated,
                 List<MavenProject> needResolve = manager.findDependentProjects(allChangedProjects);
                 needResolve.addAll(updated.keySet());
 
+                List<String> updatedPaths = updated.keySet().stream()
```

Comment:
And btw talking about your btw: if we want to be more catholic than the pope, it would be `.map(MavenProject::getProject)` :-)